webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/internal/RestApiAccounts.scala

```diff
@@ -36,24 +36,40 @@ class RestApiAccounts(
 
   serve {
     case Get("secure" :: "apiaccounts" :: Nil, req) =>
-      readApi.getAllStandardAccounts.either.runNow match {
+      implicit val prettify = restExtractor
+        .extractBoolean("prettify")(req)(identity)
+        .getOrElse(Some(false))
+        .getOrElse(
+          false
+        )
+      implicit val action: String = "getAllAccounts"
+
+      OldInternalApiAuthz.withReadAdmin(readApi.getAllStandardAccounts.either.runNow match {
```

Comment:
This could give access to tokens with full write access. Until we hash them I think it should be write here too.

Reply:
ok